src/common/img/svg.tsx

```diff
@@ -455,6 +455,23 @@ export const pencilOutlineSvg = (
   </svg>
 );
 
+export const cloneOutlineSvg = (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="16"
+    height="16"
+    viewBox="0 0 24 24"
+    fill="none"
+    stroke="currentColor"
+    strokeWidth="2.5"
+    strokeLinecap="square"
+    strokeLinejoin="round"
+  >
+    <rect x="9" y="9" width="13" height="13" rx="2" ry="2"></rect>
+    <path d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1"></path>
```

Comment:
svg fix needed just `<path.... />`

Reply:
Please check tests and build as well.

Reply:
![drafts-improvements](https://user-images.githubusercontent.com/106739598/213681876-a4a71ce1-dc25-4535-adc6-2556c7ef896e.png)